src/static/newtab.md

```diff
@@ -31,7 +31,7 @@ REPLACE_ME_WITH_THE_CHANGE_LOG_USING_SED
 -   `t`/`T` — open a URL in a new tab (`T` to pre-load current URL).
 -   `gt`/`gT` — go to the next/previous tab.
 -   `d` — close the current tab.
--   `/` — open the find search box.
+-   `/` — open the find search box. Use <kbd>ctrl</kbd> + g/G to cycle through search results.
```

Comment:
Hi. Thank you for opening this PR. We usually use `<C-g>` in order to represent chorded keyboard shortcuts. I think it'd be better to use this representation here too. Could you do that?

Reply:
Sorry, glacambre, I already merged it :(

Reply:
Argh, merged before I could comment. I'll do it myself then. Thanks again for this PR. :)